src/main/java/com/virtudoc/web/controller/DoctorsRecordsController.java

```diff
@@ -0,0 +1,8 @@
+package com.virtudoc.web.controller;
+
+public class DoctorsRecordsController {
+    @GetMapping("/doctors_records")
```

Comment:
Not a blocking issue, but wanted to open a discussion on this. Would it make more sense to combine the doctors and admins records pages into one? They have very similar content, and we can let the controller inject different data depending on who the currently logged-in user is.

Reply:
That definitely sounds like an easier approach to me, maybe I could just have all three pages consolidate into one and code the back-end in a way that displays the page based on the logged in user's role?